tests/unit/data_providers/test_pocket_hits_slate_provider.py

```diff
@@ -31,5 +31,5 @@ async def test_get_slate_reason_type(self):
     async def test_headline(self):
         slate = await self.slate_provider.get_slate()
 
-        assert slate.headline == 'Monday’s Pocket Hits'
-        assert slate.subheadline is None
+        assert slate.headline == 'Today’s Pocket Hits'
+        assert slate.subheadline == 'Monday, October 24'
```

Comment:
Do you have to do any sort of date pinning for this to not fail tomorrow?

Reply:
Yeah, freezegun is pretty awesome:
```
@freeze_time("2022-10-24 16:00:00")
```